frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-camelk.feature

```diff
@@ -12,83 +12,76 @@ Feature: CamelK Connector Event Sources
         @smoke
         Scenario: Kamelets display in event sources add page: KC-02-TC01
             Given user is at Add page
-             When user clicks on "Event Sources" card
+             When user clicks on Event Sources card
              Then user will be redirected to page with header name "Event Sources"
               And user is able to see multiple sources of the kind kamelets
 
 
-        @regression @to-do
-        Scenario: Kamelets from Community provider in event sources page : KC-02-TC02
-            Given user is at Event Sources page
-             When user clicks on "Apache Software Foundation" provider filter
-             Then user is able to see community kamelets from "Apache Software Foundation" provider
+        @regression
+        Scenario: CamelK Connector Event Source cards: KC-02-TC02
+            Given user is at developer perspective
+             When user is at Add page
+              And user clicks on Event Sources card
+             Then user will see the "AWS SQS Source" card
+              And user will see the "Salesforce Source" card
+              And user will see the "AWS Kinesis Source" card
+              And user will see the "Jira Source" card
+              And user will see the "Slack Source" card
+              And user will see the "Telegram Source" card
 
 
-        @regression @to-do
-        Scenario: CamelK Connector Event Source cards: KC-02-TC02
-            Given user is at Developer perspective
-             When user goes to +Add page
-              And user clicks on the Event Sources card
-             Then user will see the AWS 2 Simple Queue Service card
-              And user will see Salesforce card
-              And user will see AWS Kinesis card
-              And user will see Jira card
-              And user will see Slack card
-              And user will see Telegram card
-
-
-        @regression @to-do
+        @regression
         Scenario: Create AWS 2 Simple Queue Service connector: KC-02-TC03
             Given user is at Event Sources page
-             When user clicks on AWS Sqs card
-              And user clicks on Create button
+             When user clicks on the "AWS SQS Source" card
+              And user creates AWS SQS Source
```

Comment:
We can implement these by a dynamic function with switch statements.
Reference: https://github.com/openshift/console/pull/11453/files#:~:text=export%20const-,createWorkloadUsingOptions,-%3D%20(optionName